openbas-api/src/main/java/io/openbas/config/GlobalExceptionHandler.java

```diff
@@ -13,9 +13,10 @@
 public class GlobalExceptionHandler {
 
     @ExceptionHandler(ElementNotFoundException.class)
-    public ResponseEntity<ErrorMessage> handleElementNotFoundException() {
-        ErrorMessage message = new ErrorMessage("Not found");
-        return new ResponseEntity<ErrorMessage>(message, HttpStatus.NOT_FOUND);
+    public ResponseEntity<ErrorMessage> handleElementNotFoundException(ElementNotFoundException ex) {
```

Comment:
What if your exception is null?

Reply:
Is not possible, this is the purpose of this kind of handler.
https://www.baeldung.com/exception-handling-for-rest-with-spring